### DISCUSSION TITLE
bug：commit count in open source Lucene not correct

### DISCUSSION THREAD

**Discussion by wjp719** [community/community#18068] [Pull Requests]
Hi, in the [Contributors to apache/lucene · GitHub](https://github.com/apache/lucene/graphs/contributors) list, the rank shows I (wjp719) have only 2 commits, but in fact I have 6 commits in [Commits · apache/lucene · GitHub](https://github.com/apache/lucene/commits?author=wjp719). Do you know why? thanks a lot.
![image](https://user-images.githubusercontent.com/9303466/172514070-fceb07f7-c79e-46f5-a1c3-e313a5de58c8.png)

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
